src/Symfony/Component/Process/Process.php

```diff
@@ -322,7 +320,7 @@ public function start(callable $callback = null, array $env = [])
 
         if ('\\' === \DIRECTORY_SEPARATOR) {
             $commandline = $this->prepareWindowsCommandLine($commandline, $env);
-        } elseif (!$this->useFileHandles && $this->isSigchildEnabled()) {
+        } elseif ($this->isSigchildEnabled()) {
```

Comment:
```suggestion
        } elseif ('\\' !== \DIRECTORY_SEPARATOR && $this->isSigchildEnabled()) {
```

Reply:
The `elseif ` will only be considered `if ('\\' === \DIRECTORY_SEPARATOR)` (2 lines before) doesn't hit. Basically it will be checked twice but vice versa, as suggested change. 🤔